nodes/text/mn_split_text.py

```diff
@@ -17,7 +17,8 @@ class mn_SplitText(Node, AnimationNode):
 	def splitTypeChanges(self, context):
 		self.setHideProperty()
 	
-	splitType = bpy.props.EnumProperty(name = "Split Type", default = "Custom", items = splitTypes, update = splitTypeChanges)
+	splitType = bpy.props.EnumProperty(name = "Split Type", default = "Regexp", items = splitTypes, update = splitTypeChanges)
+	keepDelimiters = bpy.props.BoolProperty(default=False)
```

Comment:
yeah maybe some day. :D I also think there are tools which can do it partially. Personally I find it cleaner, but of course this has something to do with that I'm used to it
but consistency goes first